core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedValueResolver.java

```diff
@@ -775,6 +805,61 @@ private static AnnotatedValueResolver ofInjectableTypes0(String name, AnnotatedE
         };
     }
 
+    private static BiFunction<AnnotatedValueResolver, ResolverContext, Object>
+    resolverAttribute(BiFunction<ResolverContext, Class<?>, Object> getter) {
+        return (resolver, ctx) -> {
+            final Class<?> type = resolver.elementType().isPrimitive() ?
+                                  PrimitiveToReference.primitiveToReference(resolver.elementType.getName()) :
+                                  resolver.elementType();
+            // Primitive to Reference.
```

Comment:
I removed this codes because i moved this logic to `AnnotatedValueResolver#ofAttribute()`! 
Please refer to this [code](https://github.com/chickenchickenlove/armeria/blob/889d887093961d9f35be5030b3446b3cd41a61bb/core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedValueResolver.java#L643-L658).